src/main/webapp/js/common.js

```diff
@@ -943,7 +945,7 @@ function toggleSingleCollapse(e) {
     if ($(e.target).is('a') || $(e.target).is('input')) {
         return;
     }
-    var glyphIcon = $(this).find('.glyphicon');
+    var glyphIcon = $(window).find('.glyphicon');
```

Comment:
Oh right, I just realised that these functions are being called as event handlers.

:man_facepalming: